src/guides/v2.3/config-guide/deployment/pipeline/build-system.md

```diff
@@ -87,6 +87,11 @@ To set up the build system:
    ```
 
 1. Save your changes to `.gitignore` and exit the text editor.
+1. The build system should use either [default mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#default-mode) or [developer mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#developer-mode):
```

Comment:
Your newly introduced line affects the flow of existing steps that discuss the `.gitignore` file.
Please move the newly introduced line at the bottom.

Reply:
Hello @ajithkumar-maragathavel,

Thanks for your review and suggestion. The mentioned change is done.

Thanks.